main.tf

```diff
@@ -34,6 +34,7 @@ resource "aws_lb" "main" {
   internal           = var.alb_internal
   subnets            = local.alb_subnets
   security_groups    = [aws_security_group.lb.id]
+  depends_on         = [var.service_depends_on]
```

Comment:
I'll be curious to see if this works. I feel like this may cause Terraform to throw an error, since a resource is a special type (I believe, anyway).

I'll go ahead and merge and mark this as a pre-release.